doc/man1/openssl-srp.pod

```diff
@@ -65,10 +67,12 @@ adding or modifying a user.
 
 =item B<-passin> I<arg>, B<-passout> I<arg>
 
-The password source for the input and output file.
-For more information about the format of B<arg>
 see L<openssl(1)/Pass Phrase Options>.
```

Comment:
Did you mean to do this in this PR?  Shouldn't that be in another `-passin` / `-passout` PR?  Or if you meant for it to be part of this PR, aren't there more places where this change should happen?

Either way, the remaining sentence should be this:

```
See L<openssl(1)/Pass Phrase Options> for more information.
```

Reply:
No, it was an editing error.  Restored.